DXRando/DeusEx/Classes/Keypad.uc

```diff
@@ -45,6 +46,18 @@ function ToggleLocks(DeusExPlayer Player)
         bCodeKnown = True;
 }
```

Comment:
```suggestion
    UnlockDoor();
}
```

Reply:
Github won't let me put a code suggestion inside the RunEvents function because it's not part of the diff, but you get the idea.

Reply:
It unlocks the Cathedral door but it doesn't open it as vanilla does.